e2e_test/sink/sink_into_table/parallelism.slt

```diff
@@ -8,16 +8,16 @@ statement ok
 SET STREAMING_PARALLELISM TO 2;
 
 statement ok
-create table t_simple (v1 int, v2 int);
+create table t_simple (v1 int, v2 int) append only;
 
 statement ok
-create table m_simple (v1 int primary key, v2 int);
+create table m_simple (v1 int, v2 int) append only;
 
 statement ok
 SET STREAMING_PARALLELISM TO 3;
 
 statement ok
-create sink s_simple_1 into m_simple as select v1, v2 from t_simple;
+create sink s_simple_1 into m_simple as select v1, v2 from t_simple with (type = 'append-only');
```

Comment:
Change the query because if there is primary key, the original sink plan will be 
```
dev=> explain create sink s_simple_1 into m_simple as select v1, v2 from t_simple;
;
                                                QUERY PLAN                                                
----------------------------------------------------------------------------------------------------------
 StreamProject { exprs: [t_simple.v1, t_simple.v2] }
 └─StreamSink { type: upsert, columns: [v1, v2, t_simple._row_id(hidden)], downstream_pk: [t_simple.v1] }
   └─StreamExchange { dist: HashShard(t_simple.v1) }
     └─StreamTableScan { table: t_simple, columns: [v1, v2, _row_id] }
(4 rows)
```